src/uct/ib/mlx5/rc/rc_mlx5_common.h

```diff
@@ -482,6 +482,41 @@ void uct_rc_mlx5_handle_unexp_rndv(uct_rc_mlx5_iface_common_t *iface,
                                    unsigned byte_len, int poll_flags);
 
 
+static UCS_F_ALWAYS_INLINE ucs_ternary_auto_value_t
+uct_rc_mlx5_dp_ordering_ooo_get(uint8_t dp_ordering_ooo,
```

Comment:
maybe pass uct_ib_mlx5_md_t object here to use its flags?

Reply:
difficult because the flag depend on the transport (rc/dc), `dp_ordering_ooo` is either:
- `!!(md->flags & CT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_DC)`
- `!!(md->flags & CT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_RC)`

Reply:
ok